command/genesis/params.go

```diff
@@ -63,8 +63,9 @@ type genesisParams struct {
 
 	blockGasLimit uint64
 
-	baseFeeConfig       string
-	parsedBaseFeeConfig *baseFeeInfo
+	baseFeeConfig        string
+	parsedBaseFeeConfig  *baseFeeInfo
+	isBaseFeeConfigEmpty bool
```

Comment:
You don't need this bool. Just check if `baseConfig` string is an empty string instead of this bool.